operator/v1/types_csi_cluster_driver.go

```diff
@@ -61,6 +73,16 @@ const (
 // ClusterCSIDriverSpec is the desired behavior of CSI driver operator
 type ClusterCSIDriverSpec struct {
 	OperatorSpec `json:",inline"`
+	// StorageClassState determines if CSI operator should create and sync storage classes.
+	// If this field value is nil or Managed - CSI operator will continously reconcile
+	// storage class and create if necessary.
+	// If this field value is Unmanaged - CSI operator will not reconcile any previously created
+	// storage class.
+	// If this field value is Removed - CSI operator will delete the storage class it created previously.
+	// Defaults to nil
```

Comment:
We usually add a comment that explains what happens when the value is omitted
```suggestion
	// When omitted, this means the user has no opinion and the platform chooses a reasonable default,
	// which is subject to change over time.
	// The current default behaviour is Managed.
```